File: WinForms/CS/RichEditMainDemo/SpellCheckerHelper.cs
using System.Globalization;
using DevExpress.XtraSpellChecker;

namespace DevExpress.XtraRichEdit.Demos {
    public class SpellCheckerHelper {
        public static void AddDictionaries(SharedDictionaryStorage storage) {
            if(storage.Dictionaries.Count == 0)
                AddDictionaries(storage.Dictionaries);
        }
        public static void AddDictionaries(DictionaryCollection dictionaries) {
            SpellCheckerISpellDictionary dictionary = new SpellCheckerISpellDictionary(
                            DemoUtils.GetRelativePath("american.xlg"),
                            DemoUtils.GetRelativePath("english.aff"),
                            CultureInfo.GetCultureInfo("en-US")
                            );
            dictionary.CaseSensitive = true;
            dictionaries.Add(dictionary);
            SpellCheckerCustomDictionary customDictionary = new SpellCheckerCustomDictionary(
                            DemoUtils.GetRelativePath("CustomEnglish.dic"),
                            CultureInfo.GetCultureInfo("en-US"));
            dictionaries.Add(customDictionary);
        }
    }
}

File: WinForms/VB/RichEditMainDemo/SpellCheckerHelper.vb
Imports System.Globalization
Imports DevExpress.XtraSpellChecker

Namespace DevExpress.XtraRichEdit.Demos

    Public Class SpellCheckerHelper

        Public Shared Sub AddDictionaries(ByVal storage As SharedDictionaryStorage)
            If storage.Dictionaries.Count = 0 Then AddDictionaries(storage.Dictionaries)
        End Sub

        Public Shared Sub AddDictionaries(ByVal dictionaries As DictionaryCollection)
            Dim dictionary As SpellCheckerISpellDictionary = New SpellCheckerISpellDictionary(DemoUtils.GetRelativePath("american.xlg"), DemoUtils.GetRelativePath("english.aff"), CultureInfo.GetCultureInfo("en-US"))
            dictionary.CaseSensitive = True
            dictionaries.Add(dictionary)
            Dim customDictionary As SpellCheckerCustomDictionary = New SpellCheckerCustomDictionary(DemoUtils.GetRelativePath("CustomEnglish.dic"), CultureInfo.GetCultureInfo("en-US"))
            dictionaries.Add(customDictionary)
        End Sub
    End Class
End Namespace
